components/NavLink.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import cx from 'clsx';
+
+function NavLink({
+  href,
+  className,
+  activeClassName,
+  external,
+  target = '_blank',
+  rel = 'noopener noreferrer',
```

Comment:
Sorry, my last comment on this component were totally wrong.

These defaults only makes sense only when `external` is `true`, they should be to put `_blank` and `noopener noreferrer` inside the function body, otherwise it will apply to `NavLink`s even when `external` is `false`.

Reply:
In fact when `external` is set to false the target & rel won't be passed down to DOM, so either way would work I guess. will adapt initial method anyway.